LiskHQ/lisk-sdk
# Fix lisk-client export - Closes #7338

### What was the problem?

This PR resolves #7338

### How was it solved?

- main is used mainly for library, so updating the default to node.js version
- Add `browser` field to specify the usage during the browser build
- Update plugin bundles to `parcels` to read the browser field and make it easy to update the version

### How was it tested?
- Run `npm run start:web` on faucet plugin and ensure UI build is working
- Check `lisk-client/index.html` and see `lisk` is usable in the browser console
